### DISCUSSION TITLE
How to force the re-index

### DISCUSSION THREAD

**Discussion by Bidulle59** [yurijmikhalevich/rclip#137] [Q&A] [answered]
Hi,
After a few days working like a charm, it suddenly stop working.
I don't change my pictures.
It find my pictures, but give zero result.

So I can imagine that I need to re-index my directory.
How I can do this?

**Comment by yurijmikhalevich** [comment]
@Bidulle59, hi! Can you please share screenshots or a recording (maybe loom.com) showing the issue?

If you run `rclip "any query"`, it will reindex all images in the current directory (if needed).

Also, rclip will only search for images in the current directory.

E.g., if you are running it like this:
```cmd
cd C:\Users\user\Documents
rclip cat
```

It will find images of cats in the "Documents" directory (and all subdirectories). Can you check which directory you are running rclip in? Running `cd` with no arguments in Windows command line should print the current path.

**Reply by Bidulle59** [reply]
Ok, here is the link: https://www.loom.com/share/b3cee0cb44cf4725ab5d12ff2f72f658?sid=cb658bc5-a83f-41ac-b359-19977971ad7a

and my cmd line (i am on windows): cd "C:\Photos\" & rclip -t 50 "C:\Recherche\test3.jpg"
I also test with cd "C:\Photos\" & rclip -t 50 Ring
I have the same result

**Reply by yurijmikhalevich** [reply]
@Bidulle59, thank you for showing the issue! This looks wrong. I'll look into it.

rclip stores it's database under `C:/Users/<USER>/AppData/Roaming/rclip`. You can try deleting it to force it to reindex everything.

If you don't mind, can you share the `C:/Users/<USER>/AppData/Roaming/rclip/db.sqlite3` file with me? It may help me fix the issue faster. The database contains the complete filepaths and a vector representing the image features (I won't have access to the images themselves). If it's ok with you to share the database, you can send it privately via yurij@mikhalevi.ch.

**Reply by Bidulle59** [reply]
Hi,
Before your message, I have uninstall RCLIP and delete all the files in C:/Users/<USER>/AppData/Roaming/rclip
After the re-install, it works fine.

For your information, I start to use it for my work.
I am working for a jewelry distributor.
We have a big database of items that we have create.
It help to find previous models that we have make when we have a specific request

**Reply by yurijmikhalevich** [reply]
@Bidulle59, I am glad that you got it working again. And I am glad you find rclip useful for your work! :)

I'll look into what could've caused the issue when I have some time and will update you here.

**Reply by yurijmikhalevich** [reply, marked as answer]
@Bidulle59, I have found and fixed a few issues affecting index search on Windows in [rclip 2.1.6](https://github.com/yurijmikhalevich/rclip/releases/tag/v2.1.6). I am going to resolve this discussion. Let me know if you see this issue again!
